### ISSUE TITLE
Basic docs 

### ISSUE THREAD

**Issue by santos177** [BlockPo/BlockPo-to-Tradelayer#103]
Related with rpcs calls. Basic info  to show : use, syntax, expected response.

- transactions rpcs 
- info rpcs
- payloads rpcs

**Comment by patrickdugan** [comment]
I made a YAML of an API that we dind't ultimately take public, in June 2020, that maps a lot of the RPCs. I'll need to dig that out of the old laptop. But a simple .md file that extends the OmniLayer one is a good idea. I also wrote out a spec that lists them, here it is:

https://docs.google.com/document/d/1ppokG076_4TwlMnRBHWFd3iG_NvYfvIM9kx795QhMts/edit

So that is missing the new RPCs but it's a start to making this .md file and having it publicly linked to on the website ideally. Maybe not as a main header but include a link on the About page.
